avaframe/tests/data/testSim/releaseTest1_0.888_entres_dfa_ppr.asc

```diff
@@ -4,3 +4,4 @@ xllcenter     1005
 yllcenter     -5000
 cellsize      5
 NODATA_value  -9999
+1 1 1 1 1
```

Comment:
why this line?

Reply:
Because rasterio needs something in the data portion of the file, otherwise it fails